assessments/guide/security-reviewer.md

```diff
@@ -52,14 +52,41 @@ guidelines for conflict of interest situations are important to prevent:
 - SIG-Security chairs and assessment leads could intentionally or unintentionally limit the participation of an individual unfairly by asserting conflict of interest
 - Security reviews being stalled while groups belabor on who should be allowed to participate
 
-The conflicts of interest lie on a spectrum, and are classified into hard and soft conflicts.
-Hard conflicts makes a reviewer ineligible to review a project.
-A soft conflict allows a reviewer to review a project, but not as a [project lead](./project-lead.md).
-A reviewer with a soft conflict is required to document the nature of the conflict in the
-security assessment documentation. It is the responsibility of the security reviewer
-to make known his/her own conflict of interests.
-
-For each project, 2 SIG-Security chairs must sign off on the conflicts presented to them that the assessment lead has no conflicts, and reviewers have no hard conflicts. Reasons for accepting and rejecting conflicts should be documented.
+The conflicts of interest lie on a spectrum, and are classified into hard and soft conflicts:
+* A hard conflict makes a reviewer ineligible to review a project.
+* A soft conflict allows a reviewer to review a project, but not as a [project lead](./project-lead.md).
+* It is not unusual for reviewers to have soft conflicts. The diversity of reviewers that are familiar with a project and can provide a deeper insight together with a fresh set of eyes is beneficial to the success of a security review.
+
+All reviewers must provide a conflict declaration to indicate which hard or soft conflicts do, or do not exist 
+when they volunteer to be a reviewer.  This is done by placing a comment on the issue associated with the 
+assessment.
+Should a conflict arise during the time of the assessment, reviewers should notify the lead security reviewer when 
+they become aware of the potential conflict, so the new conflict may be consulted with the facilitator and/or chairs.
+
+The lead security reviewer has the responsibility of ensuring a balanced review,
+and as part of that before kicking off the assessment must:
+* Check that all reviewers have conflict-of-interest declarations,
+* Provide their own declaration of any potential conflict-of-interest (or lack thereof),
+* Ask clarifying questions, if needed
+* Assert that:
+  1. They have reviewed all conflict-of-interest declarations from the reviewers;
+  2. There are no hard-conflicts present without a waiver;
+  3. They believe that the review team is able to provide a balanced and fair assessment
+
+Update the above assertion if a new conflict-of-interest becomes known during
+the course of the assessment.
+
+The Security Assessment Facilitator or a SIG-Security chair must review the 
+Lead Security Reviewer conflict-of-interest assertion.
+
+If there any conflicts are presented, then 2 SIG-Security chairs must approve
```

Comment:
this includes soft conflicts right, this seems like it may be a potentially high overhead for chairs, since soft conflicts are likely to be common. How about making this discretionary to the security assessment facilitator

Reply:
So the way this is written, means that either the Security Assessment Facilitator Or a chair must review the entire assertion.  assertions include both hard and soft conflicts. I'll adjust the "any conflicts" to be "any hard conflicts, or multiple significant soft conflicts" for the 2 chair approval.